### DISCUSSION TITLE
Can I use unstracted offline?

### DISCUSSION THREAD

**Discussion by Taikono-Himazin** [Unstructured-IO/unstructured#2153] [Q&A] [answered]
Can I use Unstracted in offline (no internet connection)?

It is assumed that nltk.download('punkt') is running.

**Comment by AayushSameerShah** [comment, marked as answer]
Yes you can use this library offline. But note that whatever dependencies like nltk, needed to be installed have to be installed beforehand. So that time you would need the internet connection but then you are good to go for offline usage. ✌🏻

**Reply by Taikono-Himazin** [reply]
Understood. thank you!
